### ISSUE TITLE
Problem displaying selected chromsome region in brush mode

### ISSUE THREAD

**Issue by desnar** [eweitz/ideogram#91]
Hi, I'm trying to show gene mark on chrom with brush function, but I find it shows a wrong region.
I set a anno region 17:43044294-43125482 and get the result in follonig picture:
![pic](https://user-images.githubusercontent.com/7296885/35499390-0e506752-050d-11e8-8127-542a610e7517.png)
The mark is selected but the region doesn't overlap 17:43044294-43125482

Here is my html code
```html
<head>
  <script src="https://unpkg.com/ideogram@0.14.0/dist/js/ideogram.min.js"></script>
</head>
<body>

  <div>
    Selected region on human chromosome 19:
    <div><span id="from"></span>-<span id="to"></span>
      (extent: <span id="extent"></span> base pairs)
    </div>
  </div>

  <div class="ideograms-here">
    <!-- The ideogram goes here. -->
  </div>

  <script>
      function writeSelectedRange() {
        var r = ideogram.selectedRegion,
            from = r.from.toLocaleString(), // Adds thousands-separator
            to = r.to.toLocaleString(),
            extent = r.extent.toLocaleString();

        document.getElementById("from").innerHTML = from;
        document.getElementById("to").innerHTML = to;
        document.getElementById("extent").innerHTML = extent
      }
  
      var ideogram = new Ideogram({
        container: ".ideograms-here",
        organism: 'human',
        assembly: 'GRCh37',
        
        annotationsLayout: 'histogram',
        chromosomes: ['17'],
        rotatable: false,
        
        brush: true,
        onBrushMove: writeSelectedRange,
        onLoad: writeSelectedRange,
        
        orientation: "horizontal",
        showBandLabels: true, // only work in horizontal mode
        
        annotations: [{
          name: 'BRCA1',
          chr: '17',
          start: 43044294,
          stop: 43125482
        }]
      });
  </script>
</body>
```
Thanks!

**Comment by eweitz** [comment]
@desnar, thanks for reporting this issue.  I've confirmed the bug it and will update here when it's fixed.

**Comment by eweitz** [comment]
@desnar, this is fixed in the new [v0.15.0](https://github.com/eweitz/ideogram/releases/tag/v.0.15.0) release.  Please let me know if it works for you!

**Comment by desnar** [comment]
It works. Thanks!
